libstuff/STCPNode.cpp

```diff
@@ -90,22 +98,51 @@ uint64_t STCPNode::getIDByPeer(STCPNode::Peer* peer) {
     return 0;
 }
 
+STCPManager::Socket* STCPNode::acceptSocket() {
```

Comment:
This is what the old `_acceptSockets` function in BedrockServer was based on, but it's only used in STCPNode/SQLiteNode now, so it's moved here.